src/components/notificationprofile/NotificationProfileCard.test.tsx

```diff
@@ -68,25 +48,84 @@ const filters: Filter[] = [
   },
 ];
 
-const mediaOptions: { label: string; value: MediaAlternative }[] = [
-  { label: "Email", value: "EM" },
-  { label: "SMS", value: "SM" },
+const media: Media[] = [
+  { slug: "email", name: "Email" },
+  { slug: "sms", name: "SMS" },
+  { slug: "ms_teams", name: "MS Teams" },
 ];
 
-const phoneNumbers: PhoneNumber[] = [
+const destinationsArray: Destination[] = [
   {
     pk: 1,
-    user: 1,
-    // eslint-disable-next-line @typescript-eslint/camelcase
-    phone_number: "+4712345678",
+    label: "test_sms",
+    media: media[1],
+    suggested_label: "SMS: +4747474747",
+    settings: {
+      phone_number: "+4747474747"
+    }
   },
   {
     pk: 2,
-    user: 1,
-    // eslint-disable-next-line @typescript-eslint/camelcase
-    phone_number: "+4787654321",
+    label: "test_email",
+    media: media[0],
+    suggested_label: "Email: test@test.test",
+    settings: {
+      email_address: "test@test.test",
+      synced: false
+    }
   },
-];
+  {
+    pk: 3,
+    label: "test_email_synced",
+    media: media[0],
+    suggested_label: "Email: synced@test.test",
+    settings: {
+      email_address: "synced@test.test",
+      synced: true
+    }
+  },
+  {
+    pk: 4,
+    label: "test_msteams",
+    media: media[2],
+    suggested_label: "MS Teams: MS TEAMS #4",
+    settings: {
+      webhook: "https://test.test"
+    }
+  },
+]
```

Comment:
There's no guarantee that the msteams plugin is installed so you can't really test it here. We might need a discussion about this.

We could have a dummy plugin that has a webhook?